consoles/sshVirtsh.pm

```diff
@@ -631,7 +631,14 @@ sub run_cmd {
     my ($self, $cmd, %args) = @_;
     my %credentials = $self->get_ssh_credentials($args{domain});
     delete $args{domain};
-    return $self->backend->run_ssh_cmd($cmd, %credentials, %args);
+    my @response;
+    if ($args{wantarray}) {
+        @response = $self->backend->run_ssh_cmd($cmd,  %credentials, %args);
+        return @response;
+    } else {
+        $ret = $self->backend->run_ssh_cmd($cmd, %credentials, %args);
+        return $ret
+    }
```

Comment:
I don't see, that your change is solving the problem. 
I would go with:

```suggestion
    if ($args{wantarray}) {
        return [$self->backend->run_ssh_cmd($cmd,  %credentials, %args)];
    } else {
        return $self->backend->run_ssh_cmd($cmd, %credentials, %args);
    }
```

Reply:
then please use the ternary operator directly:

```suggestion
    return $args{wantarray} ? @{$self->backend->run_ssh_cmd($cmd,  %credentials, %args)} : $self->backend->run_ssh_cmd($cmd, %credentials, %args);
```